content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -27,20 +27,17 @@ Our current process is as follows:
 
   - Teammates can **_reorder_ proposed issues** on the ["Next iteration" board](https://github.com/orgs/sourcegraph/projects/200/views/4) before the mid-iteration sync to their liking. The order at the time of the mid-iteration Monday sync is the _proposed order_.
 
-  - At the mid-iteration [Monday sync](#weekly-sync), we take a look at the proposed issues together on the ["Next iteration" view](https://github.com/orgs/sourcegraph/projects/200/views/4) to **reach agreement** on the **set** of iteration issues, their **assignees**, and **order**, taking into account our [goals and roadmap](../../../../../strategy-goals/strategy/code-graph/code-insights/index.md), technical aspects, estimates, workloads on individuals, and release dates (the 20th of every month).<br>
-    As a rule of thumb, the sum of estimates for each individual should not exceed **8 days** (out of a 10-work-day iteration) so we have enough buffer, time to reply to asks from other teams, to plan the next iteration's projects, and self-organized slack time.
-  - On the Thursday before the next iteration begins, we have a brief synchronous meeting to finalize our planning for the iteration. Depending on our needs, this meeting may cover: questions about priority, clarity, or edge cases of specific features for the next iteration; refining plans for the following Monday iteration start; surfacing any missing GitHub issues (for known items) ahead of the Monday kickoff so they can be made before Monday. This meeting is also an opportunity to review the status of our current iteration and examine if any issues should be carried over to the next iteration. This meeting is in **experimental** stage – we will try this process through mid-February 2022 and then decide if it's consistently worth the time.
+  - On the **Thursday before the next iteration begins**, we have a brief synchronous meeting to finalize our planning for the iteration on the ["Next iteration" view](https://github.com/orgs/sourcegraph/projects/200/views/4) that we conducted asynchronously as described above. Depending on our needs, this meeting may cover: questions about priority, clarity, or edge cases of specific features for the next iteration; refining plans for the following Monday iteration start; surfacing any missing GitHub issues (for known items) ahead of the Monday kickoff so they can be made before Monday. This meeting is also an opportunity to review the status of our _current_ iteration and examine if any issues should be carried over to the next iteration. This meeting is in **experimental** stage – we will try this process through mid-February 2022 and then decide if it's consistently worth the time.
```

Comment:
It is currently mid-February 2022. Is this meeting still experimental and we should move this date out? Or should we just delete this line altogether?

Reply:
Good point, unless somebody objects I'm gonna remove the line. cc @Joelkw